org.opent2t.sample.lamp.superpopular/com.hue.bulb/js/colour.js

```diff
@@ -0,0 +1,359 @@
+"use strict";
+
+const ChangeTolerance = 0.0001;
+const MaxHue = 360.0;
+const MaxColour = 255;
+
+/*
+ * Method to see if the given XY value is within the reach of the lamps.
+ *
+ * @param xyColour is a two-element object representing a point containing the X,Y value
+ * @param colourGamut   the color gamut of the light device.
+ * @return true if within reach, false otherwise.
+ */
+function isPointInLampsReach(xyColour, colourGamut){
```

Comment:
Sure but here i wanted to reference the spec specifically for the spelling and why we chose the British version :) - a simple comment here explainng the use of color vs colour should suffice - it's a nit by the way :) i will not block checkin because of it..something i was wondering when i saw the code

---
In reply to: [94528610](https://github.com/openT2T/translators/pull/143#discussion_r94528610) [](ancestors = 94528610,94481441,94470752)